Clojure/Comment.tmPreferences

```diff
@@ -4,7 +4,7 @@
 	<key>name</key>
 	<string>Comment</string>
 	<key>scope</key>
-	<string>source.clojure</string>
```

Comment:
This works fine, but note that _usually_ commas are used unless there is a multilevel command that needs an ORed context.

Reply:
I will get that fixed.

Reply:
Probably also in keymap? I don't often map keys, so I'm less familiar with those.

Reply:
I updated both of them to use `,` instead of `|`.